src/main/java/com/minecolonies/coremod/entity/ai/citizen/herders/AbstractEntityAIHerder.java

```diff
@@ -494,10 +494,22 @@ private void butcherAnimal(final EntityAnimal animal)
         }
     }
 
+    /**
+     * Gets an ItemStack of breedingItem for 2 animals.
+     *
+     * @return the BreedingItem stack.
+     */
+    public ItemStack getBreedingItems()
+    {
+        final ItemStack breedingItem = getBreedingItem();
```

Comment:
Call Copy on the stack to clone it as well.
On the other Hand you set the stacksize not using ItemStackUtils.